src/scheduling/appointments/new/NewAppointment.tsx

```diff
@@ -42,6 +42,11 @@ const NewAppointment = () => {
 
   const onNewAppointmentSaveSuccess = (newAppointment: Appointment) => {
     history.push(`/appointments/${newAppointment.id}`)
+    Toast(
+      'success',
+      t('Success!'),
+      `${t('scheduling.appointment.successfullyCreated')} ${newAppointment.id}`,
+    )
```

Comment:
Could we add a test to make sure that the toast is getting called and has the proper message?